From 269ef64fc327796fc6c5a8220e817bbe09d5b2cc Mon Sep 17 00:00:00 2001
From: Anamika Lal <anamikax.lal@intel.com>
Date: Mon, 7 Aug 2017 16:33:00 +0530
Subject: [PATCH 4269/4301] ASoC: Intel: FPGA: Fix build error for mips

commit 9ddc97060c4f686547b91bb605da8ace0de20565 from
git://git@github.com:01org/linux-intel-4.9.git

Below build regression observed for mips with allmodconfig:

sound/soc/intel/fpga/hda_intel_cnl_fpga.c:277:1: error: type defaults
to 'int' in declaration of 'module_pci_driver' [-Werror=implicit-int]

To fix this, x86 dependency added to SND_SOC_INTEL_CNL_FPGA.

Change-Id: Ia975415cafad536832d3383ed3e8c4314bf0d301
Signed-off-by: Anamika Lal <anamikax.lal@intel.com>
Reviewed-on: https://git-gar-1.devtools.intel.com/gerrit/17517
Reviewed-by: Diwakar, Praveen <praveen.diwakar@intel.com>
Reviewed-by: Singh, Guneshwor O <guneshwor.o.singh@intel.com>
Reviewed-by: Koul, Vinod <vinod.koul@intel.com>
Tested-by: Sm, Bhadur A <bhadur.a.sm@intel.com>
---
 sound/soc/intel/Kconfig |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/sound/soc/intel/Kconfig b/sound/soc/intel/Kconfig
index 92d9582..ea19827 100644
--- a/sound/soc/intel/Kconfig
+++ b/sound/soc/intel/Kconfig
@@ -273,6 +273,7 @@ config SND_SOC_INTEL_SKL_RT286_MACH
 
 config SND_SOC_INTEL_CNL_FPGA
 	tristate
+	depends on X86
 	help
 	  Say Y here to enable the support for CNL audio FPGA Device.
 
-- 
1.7.5.4

